services/ui-src/src/page/temporary-extension/TemporaryExtensionForm.tsx

```diff
@@ -23,7 +18,7 @@ const temporaryExtensionFormInfo: OneMACFormConfig = {
   idFormat: idFormat,
   actionsByStatus: Workflow.defaultActionsByStatus,
   raiLink: ROUTES.WAIVER_RAI,
-  landingPage: ONEMAC_ROUTES.PACKAGE_LIST_WAIVER,
+  landingPage: ROUTES.DETAIL + "/parentType/parentId#temp-extension",
```

Comment:
I think I understand why you are doing things this way, though I believe we do *have* the location state when this component is called.  But... if we are trying for a "config file/execution file" pattern, this does help keep executable code out of this file...

Anyway, it is worth trying and should not be too terrible to change later if it causes problems :)